docs/supported-sources/zendesk.md

```diff
@@ -0,0 +1,84 @@
+# Zendesk
+
+[Zendesk](https://www.zendesk.com/) is a cloud-based customer service and support platform. It offers a range of features including ticket management, self-service options, knowledgebase management, live chat, customer analytics, and conversations.
+
+ingestr supports Zendesk as a source.
+
+The Zendesk supports two authentication methods:
+
+- OAuth Token
+- API Token
+
+All resources supports API Token except "chats resources" i.e Zendesk Chat API.
+
+## URI Format
+
+The URI format for Zendesk is as follows:
+
+```plaintext
+zendesk://<sub-domain>?oauth_token=<oauth_token>
+```
+
+```plaintext
+zendesk://<sub-domain>?api_token=<api_token>&email=<email>
+```
+
+URI parameters:
+
+- `subdomain`: Unique zendesk subdomain that can be found in account url
+- `email`: Email address of the user
+- `api_token`: API token used for authentication with zendesk
+- `oauth_token` : OAuth token used for authentication with zendesk
+
+## Setting up a Zendesk Integration
+
+Zendesk requires a few steps to set up an integration, please follow the guide dltHub [has built here](https://dlthub.com/docs/dlt-ecosystem/verified-sources/zendesk#setup-guide).
+
+Once you complete the guide, if you decide to use an OAuth Token, you should have a subdomain and an OAuth token. Let’s say your subdomain is `mycompany` and your OAuth token is `qVsbdiasVt`.
+
+```sh
+ingestr ingest --source-uri "zendesk://mycompany?oauth_token=qVsbdiasVt" \
+--source-table 'tickets' \
+--dest-uri 'duckdb:///zendesk.duckdb' \
+--dest-table 'zendesk.tickets' \
+--interval-start '2024-01-01'
+```
+
+If you decide to use an API Token, you should have a subdomain, email, and API token. Let’s say your subdomain is `mycompany`, your email is `john@get.com`, and your API token is `nbs123`.
+
+```sh
+ingestr ingest --source-uri "zendesk://mycompany?email=john@get.com&api_token=nbs123" \
+--source-table 'tickets' \
+--dest-uri 'duckdb:///zendesk.duckdb' \
+--dest-table 'zendesk.tickets' \
+--interval-start '2024-01-01'
+```
+
+The result of this command will be a table in the `zendesk.duckdb` database
+
+## Available Tables
+
+Zendesk source allows ingesting the following sources into separate tables:
```

Comment:
could you please add docs to the Zendesk API so that people understand what these different API endpoints refer to in Zendesk's concepts?